### PR TITLE
 Add action plugin to support dnos6_config module

### PR DESCRIPTION
**Pull request by skg-net**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME
- modules/core/network/dnos6/dnos6_config
##### ANSIBLE VERSION
- ansible 2.2.0 (devel a1947401ba) last updated 2016/09/08 17:44:28 (GMT -700)
  lib/ansible/modules/core: (detached HEAD db38f0c876) last updated 2016/09/08 17:44:58 (GMT -700)
  lib/ansible/modules/extras: (detached HEAD 8bfdcfcab2) last updated 2016/09/08 17:44:58 (GMT -700)
  config file = /home/skg/lothlorien/ansible.cfg
  configured module search path = Default w/o overrides
##### SUMMARY
- Add action plugin to support dnos6_config module

### CODE REVIEW

**Comment by skg-net** [comment]
@privateip @gundalow, please have a look at it.

**Comment by skg-net** [comment]
ready_for_review

File: lib/ansible/plugins/action/dnos6_config.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2015 Peter Sprygada <psprygada@ansible.com>
```
Comment (gundalow) [inline lib/ansible/plugins/action/dnos6_config.py:2]: Needs changing to you

**Comment by gundalow** [comment]
needs_revision

**Comment by gundalow** [comment]
`CHANGELOG.md` needs rebase then I can merge this

**Comment by gundalow** [comment]
Merged, thanks

**Pull request merged**